src/go/k8s/tests/e2e/pandaproxy-produce-consume-tls-client-external-ca/00-redpanda-cluster.yaml

```diff
@@ -0,0 +1,71 @@
+# The CA cert "provided" by the customer.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ca-cert
+  labels:
+    app.kubernetes.io/component: redpanda
+    app.kubernetes.io/name: redpanda
+    app.kubernetes.io/instance: cluster-proxy
+  annotations:
+    operator.redpanda.com/external-ca: "true"
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUR1RENDQXFBQ0NRRDRTZ0RKTDVvQ1dUQU5CZ2txaGtpRzl3MEJBUXNGQURDQm5URUxNQWtHQTFVRUJoTUMKVlZNeEN6QUpCZ05WQkFnTUFrTkJNUll3RkFZRFZRUUhEQTFUWVc0Z1JuSmhibU5wYzJOdk1SY3dGUVlEVlFRSwpEQTVTWldSd1lXNWtZU3dnU1c1akxqRU9NQXdHQTFVRUN3d0ZRMnh2ZFdReEd6QVpCZ05WQkFNTUVtTnNiM1ZrCkxuSmxaSEJoYm1SaExtTnZiVEVqTUNFR0NTcUdTSWIzRFFFSkFSWVVkR1Z6ZEdzNGMwQnlaV1J3WVc1a1lTNWoKYjIwd0hoY05Nak13TXpFMU1qRXdOelV3V2hjTk16TXdNekV5TWpFd056VXdXakNCblRFTE1Ba0dBMVVFQmhNQwpWVk14Q3pBSkJnTlZCQWdNQWtOQk1SWXdGQVlEVlFRSERBMVRZVzRnUm5KaGJtTnBjMk52TVJjd0ZRWURWUVFLCkRBNVNaV1J3WVc1a1lTd2dTVzVqTGpFT01Bd0dBMVVFQ3d3RlEyeHZkV1F4R3pBWkJnTlZCQU1NRW1Oc2IzVmsKTG5KbFpIQmhibVJoTG1OdmJURWpNQ0VHQ1NxR1NJYjNEUUVKQVJZVWRHVnpkR3M0YzBCeVpXUndZVzVrWVM1agpiMjB3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLQW9JQkFRQ2VseUdxZHoyMi83dEZRV3RRCkRBTjNtTkI4VGVyUWMyWmpyMHl2dStFVWdvdnBoaTVOQUU0TDRwblFFUFZsak1HZDdtL0gwQm1nWElHdTJxMmQKb05GOW1WUURtWlI3cVJSNkZlYy95SVRjU1pYem1wTUZsQW10YUZrUXJkekJ4MUpGTHBFa1VkVkVTNlA3RWZvUQp5cWlhd1dBTmswNVdEMnJvYnBhWlNFRFlBTmVTdGZobXdaOENiaGRpTGtBUW9MekxzUjAvL3ZvTXgvT1dkRHJuCnQ4bWxQNnR1emFIcW1iV2ZyWW5DbWVzQmY4N0RzWXEvYm1JTlYyU2E3TndJUDAxZm5EMnU4VEh1bkNBOTdWa3AKbWlEOHpwMlRCNzRaQzhwRERoSjlzNldRbmNPcDdoSC9WcllmaFZHVjl3N282UmNBS3k4OU16S3Y5eGtyK0JNNwpXeFRYQWdNQkFBRXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBQ2dHZ29hRlI1OHlPWC9Wenc2L0tpQzJYaURFCnQvZnU1RVZ3WlVodEwxY2NYVGt0clJUdExWNjJKck1yTFBXb1hUc2Z6RzRiWHdnem4yY2hHcUNsSGM0dnpmMzcKbG1sR0wzUGxJVUU5M056b3hJSk9ER2VlenV5UnAxRUcxaFVwaloweHdTQWpITFJ5THdwNGdZRVNQcU1RbDlzUgpVS2VPUjN4aGxkSnFnLzM1MENXWWpnTVJyNCtNUVNtSnZzaVVoRTVmU3JRTHI5c1dUMjIzaDBJQ1crTW1uL2xNCkFNNlRJc0lQRHIzVjFqWXFFeTlnQ3IrWXRJa21SREx6VFpzZzhjazU5NG5KdDVxUDAzY3BTSStmRDBZS3JUMEMKMFJ1cnc5ZStZUlpyMy9rRFJNSnM5UUpkY056cDQ1akhjZ2JudDQySWJzcklPbFlGekJua0RoQy9sK3M9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
+
+---
+
+# The key and client cert that was generated using the CA above. Used by later tests.
+# !!!!!! NOTE !!!!!!: The cert will expires on Mar 16 22:57:08 2026 GMT
```

Comment:
You can use TestStep to generate self sign certificate using openssl cli.

Example definition
https://github.com/redpanda-data/redpanda/blob/a628e95cf23f1d60a29b2d7313f688ca975cac74/src/go/k8s/tests/e2e/additional-configuration/01-verify-config.yaml

When you have self sign cert you can create secret using `kubectl create secret client-cert --from-file client.key --from-file client.crt`

That way we don't need to maintain this test. Even if it will expire in 3 years (2026).

Reply:
Sounds good. Done.